packages/datadog_flutter_plugin/lib/src/logs/ddlogs_web.dart

```diff
@@ -84,7 +86,19 @@ class DdLogsWeb extends DdLogsPlatform {
   ) async {
     final logger = _activeLoggers[loggerHandle];
     final webLogLevel = _toWebLogLevel(level);
-    logger?.log(message, valueToJs(attributes, 'attributes'), webLogLevel);
+    JSError? error;
+    if (errorMessage != null || errorKind != null) {
+      error = JSError();
+      error.stack = convertWebStackTrace(errorStackTrace);
+      error.message = errorMessage;
+      error.name = errorKind ?? 'Error';
+    }
+    logger?.log(
+      message,
+      valueToJs(attributes, 'attributes'),
+      webLogLevel,
+      error,
```

Comment:
error can be uninitialized here if `if` block above is not executed? what will be the behavor in this case, will we have a crash?

Reply:
(And this case should be handled in the integration tests)